src/main/java/com/google/sps/utils/User.java

```diff
@@ -0,0 +1,57 @@
+  package com.google.sps.utils;
+  
+  
+  /** A user and the user's current score */
+  public final class User {
+    String userID;
+    String userName;
+    int score;
+    public static final String SESSION_ID_COOKIE_NAME = "SessionID";
+    long quiz_timing;
+    String blobKey;
+    String gameId;
+ 
+    public User(String userID, String userName, int score) {
+        this.userID = userID;
+        this.userName = userName;
+        this.score = score;
+    }
+
+    public String getId(){
+        return userID;
+    }
+    public String getName(){
+        return userName;
+    }
+    public long getScore(){
+        return score;
+    }
+    public long getQuizTiming(){
+        return quiz_timing;
+    }
+    public String getBlobKey(){
+        return blobKey;
+    }
+    public String getGameId(){
+        return gameId;
+    }   
+
+    public void setId(String userID){
+        userID = userID;
+    }
+    public void setName(String userName){
```

Comment:
this too -- u really shouldn't be able to change your name once the user entity is created